README.md

```diff
@@ -0,0 +1,20 @@
+# Carousel
+Displays images in a carousel.
+
+## Features
+* Set static images in the modeller
+
+## Dependencies
+Mendix 6: Tested on 6.8.0.
```

Comment:
The fact that something is tested in a version doesn't give any guarantees that it actually works. Vice versa if something isn't tested in a certain version also doesn't mean it doesn't work. In other words, it doesn't mean much. What customers care about is the support. For which versions is this widget supported? That we don't know yet.